entrypoint.d/40-addons-link

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import logging
+import os
+import sys
+
+import odoobaselib
+
+if not odoobaselib.LINK:
```

Comment:
`LINK` is currently a build arg, so it will never be there at run time. You need to change that in the Dockerfile and possibly in the docs.

However, thinking about it twice, if we move the symlinking process to the entrypoint, then it makes not much sense to add the feature of *not* linking (this made sense when we had the problem said before, but not with this patch...). We can just remove this (remember: here, in Dockerfile and in README if it was there).

I love removing things! 😊